app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js

```diff
@@ -42,7 +48,7 @@ export default class extends Controller {
     );
 
     this.model
-      .save()
+      .save(this.saveAttrNames)
```

Comment:
Is it the only preferences "page" that has this issue? 🤔

Reply:
Yes, I could not find any other.